### DISCUSSION TITLE
url are sensitive

### DISCUSSION THREAD

**Discussion by yeganehaym** [filebrowser/filebrowser#1180] [Q&A] [answered]
**Description**
A clear and concise description of what the issue is about. What are you trying to do?
access files by url in my app

**Expected behaviour**
What did you expect to happen?
load files or images

**What is happening instead?**
Please, give full error messages and/or log.
path is case sensitive
xx.com/Resource/Dealers/1.jpg not coming
xx.com/Resources/Dealer/1.jpg coming up

**Additional context**
Add any other context about the problem here. If applicable, add screenshots to help explain your problem.

**How to reproduce?**
Tell us how to reproduce this issue. How can someone who is starting from scratch reproduce this behaviour as minimally as possible?

**Files**
A list of relevant files for this issue. Large files can be uploaded one-by-one or in a tarball/zipfile.

**Comment by o1egl** [comment]
I'm not sure what do you mean. Your url examples are different. Please give more context.

**Comment by yeganehaym** [comment]
sorry i made a mistake in example
look at these
xx.com/Resource/Dealers/1.jpg coming
xx.com/Resource/dealers/1.jpg not coming
dealers and Dealers

**Comment by o1egl** [comment, marked as answer]
It is an expected behaviour. Some of the filesystems are case insensitive and others are not. Which means you could possible have `Dealers` and `dealers` folders at the same time.
